<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>To_Do</class>
 <widget class="QDialog" name="To_Do">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_name">
   <property name="geometry">
    <rect>
     <x>92</x>
     <y>20</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
font: 700 14pt &quot;Segoe UI&quot;;
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>540</y>
     <width>83</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Update</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>80</y>
     <width>291</width>
     <height>431</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLineEdit" name="lineEdit_toDo1"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_toDo2"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_toDo3"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_toDo4"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_toDo5"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_toDo6"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
